app/models/card.rb

```diff
@@ -1,16 +1,22 @@
 class Card < ActiveRecord::Base
   belongs_to :user
+  belongs_to :category
 
   mount_uploader :avatar, AvatarUploader
 
   scope :created_before, -> (data) { where("review_date <= ?", data) }
+  scope :from_active_category, -> { joins(:category).where('categories.active = ?', true) }
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.